base/linalg/dense.jl

```diff
@@ -29,6 +35,41 @@ function scale!(X::Array{T}, s::Real) where T<:BlasComplex
     X
 end
 
+
+function isone(x::StridedMatrix)
```

Comment:
In this file, most matrices have an upper-case name, like `A` or `M`, could be good to update the name `x` here.